metricbeat/docs/modules_list.asciidoc

```diff
@@ -10,26 +10,28 @@ This file is generated! See scripts/mage/docs_collector.go
 |<<metricbeat-module-apache,Apache>>     |image:./images/icon-yes.png[Prebuilt dashboards are available]    |  
 .1+| .1+|  |<<metricbeat-metricset-apache-status,status>>   
 |<<metricbeat-module-aws,aws>>     |image:./images/icon-yes.png[Prebuilt dashboards are available]    |  
-.6+| .6+|  |<<metricbeat-metricset-aws-cloudwatch,cloudwatch>> beta[]  
+.6+| .6+|  |<<metricbeat-metricset-aws-cloudwatch,cloudwatch>> beta[]
```

Comment:
Nit: Not sure why it created this diff. Windows? I would not expect this to happen, as this file is generated. So you perhaps opened it manually and the IDE did some magic?

Reply:
bad resolve of conflicts, should look ok now